mappings/net/minecraft/predicate/block/BlockStatePredicate.mapping

```diff
@@ -3,7 +3,9 @@ CLASS net/minecraft/class_2715 net/minecraft/predicate/block/BlockStatePredicate
 	FIELD field_12420 manager Lnet/minecraft/class_2689;
 	FIELD field_12421 propertyTests Ljava/util/Map;
 	METHOD <init> (Lnet/minecraft/class_2689;)V
-		ARG 1 manager
+		ARG 1 stateManager
+	METHOD apply test (Ljava/lang/Object;)Z
```

Comment:
```suggestion
	METHOD apply apply (Ljava/lang/Object;)Z
```
Don't rename methods which name isn't obfuscated, it creates conflicts in the mappings